index.html

```diff
@@ -2455,113 +2455,129 @@ <h2>Services</h2>
-implementation MUST NOT convert between <a>representations</a> without first parsing to
-a <a href="#data-model">data model</a> (described in Sections <a
-href="#data-model"></a> and <a href="#core-properties"></a>); translation
-between any two <a>representations</a> is done by parsing the source <a>representation</a>
-into the <a href="#data-model">data model</a> and then serializing the <a
-href="#data-model">data model</a> into the target <a>representation</a>.
+A concrete serialization of any data structure in this specification is called a
+<a>representation</a>. A <a>representation</a> is created by serializing the <a
+href="#data-model">data model</a> through a process called
+<dfn>production</dfn>. A <a>representation</a> is transformed back into the <a
+href="#data-model">data model</a> through a process called
+<dfn>consumption</dfn>. The <em>production</em> and <em>consumption</em> process
+enables the conversion of information from one <a>representation</a> to another.
+Although this specification defines <a>representations</a> for JSON, JSON-LD,
+and CBOR, developers can use any other <a>representation</a>, such as XML or
+YAML, that is capable of expressing the <a href="#data-model">data model</a>.
+The following sections define the general rules for <a>production</a> and
+<a>consumption</a> and the JSON, JSON-LD, and CBOR <a>representations</a>.
     </p>
 
-    <p>
-Although syntactic mappings are provided for JSON, JSON-LD, and CBOR here,
-applications and services MAY use any other data <a>representation</a> syntax that is
-capable of expressing the <a href="#data-model">data model</a>, such as XML or
-YAML.
-    </p>
-
-    <p>
-Producers MUST indicate which <a>representation</a> of a document has been used via a
-media type in the document's metadata. Consumers MUST determine the
-<a>representation</a> of a <a>DID document</a> via the <code>contentType</code> <a>DID
-resolver</a> metadata field (see <a href="#did-resolution"></a>), <em>not</em>
-through the content of the <a>DID document</a> alone.
-    </p>
+    <section>
+      <h2>Production and Consumption</h2>
 
-    <p>
-Unrecognized entries in the <a href="#data-model">data model</a> MUST be
-preserved. An unrecognized entry is any entry that does not have explicit
-processing rules known to the consumer or producer. Consumers MUST add all
-entries that do not have explicit processing rules for the
-<a>representation</a> being consumed to the <a href="#data-model">data
-model</a> using only the <a>representation</a>'s generic type processing rules.
-Producers MUST serialize all entries in the <a href="#data-model">data
-model</a> that do not have explicit processing rules for the
-<a>representation</a> being produced using only the <a>representation</a>'s
-generic type processing rules.
-    </p>
+      <p>
+The rules in this section apply to all implementations seeking to be compatible
+with independent implementations of the specification. Deployments of this
```

Comment:
This language sounds funny to me, not sure exactly what it means :)

Reply:
This was language that @jricher created, I believe... I think what he's saying is "If you are writing an implementation, and you want to be compatible with other implementations, these rules apply to you."

I do agree that the sentence is hard to read and that we don't really need it. Let's see if @jricher has alternative language, and if not, I can come up with something (or just remove the language, since it's fairly obvious that the rules are there in order to achieve interoperability).